### ISSUE TITLE
KeyError after opening a manga directory with UTF-8 characters

### ISSUE THREAD

**Issue by SoakedTofu** [blueaxis/Poricom#28]
Hi. First of all, thanks a lot for developing this app.

With the new 0.4.0 release, I'm having the following issue: if I open a manga directory, close the app and then reopen it, I end up with the following error. The program won't open again, so I have to replace it with a new copy.

![Screenshot_20230108_125109](https://user-images.githubusercontent.com/95464631/211211232-1c27f383-d8c1-4a73-a35d-92116da296b0.png)

I'm running the program through Wine on Linux, but I also tested the issue in a Windows 11 VM and it also happens.

**Comment by bluaxees** [comment]
Unfortunately, I am unable to release a fix until the weekend. For now, instead of redownloading the app, download `config.toml` [here](https://github.com/bluaxees/Poricom/blob/main/code/utils/config.toml) and replace `app/utils/config.toml` in the app directory.

Also, can you tell me what happens to `app/utils/config.toml` when you reopen the app?

**Comment by SoakedTofu** [comment]
Hi, I didn't build it myself, it's the 0.4.0 release. I just checked the config.toml and it goes completely blank after the error, but I think I found the issue after some testing. I think the app doesn't like if the manga directory contains non-Latin, i.e. Japanese characters in it's save path. At least, that's my guess, because when I changed the directory of my manga from よつばと! 第01巻 to Yotsubato vol.1, the app stopped crashing afterwards.
